backend/apps/chats/views.py

```
@@ -530,6 +656,15 @@ def messages(self, request, pk=None):
 
 		residence = getattr(request, "residence", None)
 		if residence:
+			# Incluir participantes para que consumidores (ej. admin) puedan filtrar notificaciones
+			participants = []
+			try:
+				participants = [
+					conv.member_one.user.email if conv.member_one and getattr(conv.member_one, 'user', None) else None,
+					conv.member_two.user.email if conv.member_two and getattr(conv.member_two, 'user', None) else None,
+				]
+			except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)